src/client-store/account.proto

```diff
@@ -0,0 +1,27 @@
+/// Type: Client side helper model
+
+syntax = "proto3";
+package dvote.types.v1;
+
+import "metadata/entity.proto";
+import "client-store/key.proto";
+
+// Helper to serialize and store a set of entries
+message AccountsStore {
+    repeated Account items = 1; 
+}
+
+message Account {
```

Comment:
Ok, I was under the impression that the `peers` and `identityId` flag were not being used. 
`identityId`: since it is redundant i would avoid replicate it
`peers`: seems a bit counter intuitive name for what is being used. would you have another proposal?

To be honest, in my first draft i separated the app account with the web account but it seemed exagerrated. Maybe you find it reasonable. It would be something like:
```
message WebAccountsStore {
    repeated WebAccount items = 1; 
}

message AppAccountsStore {
    repeated AppAccount items = 1; 
}

message AppAccount {
   Account account =1;
   string version = 2;
   string analyticsId = 3;
   
   // Pointer to fetch the subscribed entities
    message Peers {
        repeated EntityReference entities = 1;
        repeated PeerIdentity identities = 2;  // external identities that we have interacted with
    }
    Peers peers = 5; 

   ...
}

message PeerIdentity {
    string alias = 1;
    string identityId = 2; 
}

message WebAccount {
   Account
}

message Account {
   ....
}

```

Then the backup model wanted to retrieve the account it could receive in a variable either an AppAccount or a WebAccount.